AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/StorageEngineAdapter+SQLite.swift

```diff
@@ -16,12 +16,11 @@ final class SQLiteStorageEngineAdapter: StorageEngineAdapter {
 
     internal var connection: Connection!
     private var dbFilePath: URL?
+    static let dbVersionKey = "com.awsamazon.dataStore.dbVersion"
```

Comment:
Changed, but in PictureThis the bundle identifier is "com.awsamazon.PictureThis" instead of "com.amazonaws.PictureThis". I guess no matter "awsamazon" or "amazonaws" doesn't matter?

Reply:
oh good point, not sure which is the right one, more important for the codebase to be consistent regardless of PictureThis's bundle since it's external sample app